utils/potential_utf/src/uchar.rs

```diff
@@ -0,0 +1,324 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
I was going to say "great job with the docs, comments, and test coverage" but then I realized that this code is moved from somewhere else :)